help/destinations/catalog/crm/salesforce.md

```diff
@@ -56,35 +56,58 @@ Also ensure that the [scopes](https://help.salesforce.com/s/articleView?id=conne
 * ``refresh_token``
 * ``offline_access``
 
-#### Create custom field within Salesforce {#prerequisites-custom-field}
+Finally, ensure that the `password` grant is enabled within your [!DNL Salesforce] account. Refer to the [!DNL Salesforce] [OAuth 2.0 Username-Password Flow for Special Scenarios](https://help.salesforce.com/s/articleView?id=sf.remoteaccess_oauth_username_password_flow.htm&type=5) documentation if you need guidance.
 
-Create the custom field of type `Text Area Long`, which Experience Platform will use to update the segment status within [!DNL Salesforce CRM]. 
-Refer to the Salesforce documentation to [create custom fields](https://help.salesforce.com/s/articleView?id=sf.adding_fields.htm&type=5) if you need additional guidance. 
+>[!IMPORTANT]
+>
+>If your [!DNL Salesforce] account administrator has restricted access to trusted IP ranges, you need to contact them to get Platform IP's allowed. Refer to the [!DNL Salesforce] [Restrict Access to Trusted IP Ranges for a Connected App](https://help.salesforce.com/s/articleView?id=sf.connected_app_edit_ip_ranges.htm&type=5) documentation if you need additional guidance.
```

Comment:
I think this would be a good place to link out to the Experience Platform IPs page, like: 

[...] you need to contact them to get [Experience Platform IPs](https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/streaming/ip-address-allow-list.html?lang=en) allowlisted.